hail_search/hail_search_query.py

```diff
@@ -658,6 +677,32 @@ def _format_transcript_args(self):
         })
         return args
 
+    def _get_family_passes_quality_filter(self, quality_filter, ht=None, pathogenicity=None, **kwargs):
+        passes_quality = super(VariantHailTableQuery, self)._get_family_passes_quality_filter(quality_filter)
+        clinvar_path_ht = False if passes_quality is None else self._get_clinvar_filter_ht(pathogenicity)
+        if not clinvar_path_ht:
+            return passes_quality
+
+        return lambda entries: hl.is_defined(clinvar_path_ht[ht.key]) | passes_quality(entries)
```

Comment:
The `ht` here can be `None`. So `ht.key` can fail.

Reply:
it can never be None, its just a function of how overriding keyword args works

Reply:
Then you may want to remove the default value for `ht` from the params.